x-pack/plugins/observability_solution/synthetics/server/routes/monitor_cruds/edit_monitor.ts

```diff
@@ -272,7 +273,11 @@ export const syncEditedMonitor = async ({
       server.telemetry,
       formatTelemetryUpdateEvent(
         editedMonitorSavedObject as SavedObjectsUpdateResponse<EncryptedSyntheticsMonitorAttributes>,
-        decryptedPreviousMonitor,
+        {
+          ...decryptedPreviousMonitor.attributes,
+          id: decryptedPreviousMonitor.id,
```

Comment:
On attributes, id is the monitor query id.

```suggestion
          config_id: decryptedPreviousMonitor.id,
          id: decryptedPreviousMonitor.attributes.id
```